cli/src/com/redhat/ceylon/common/tools/AbstractTestTool.java

```diff
@@ -92,9 +92,11 @@ public void setVersion(String version) {
         this.version = version;
     }
     
-    @Option(longName = "tap")
-    @Description("Enables the Test Anything Protocol v13.")
-    public void setTap(boolean tap) {
+    @OptionArgument(argumentName = "file")
```

Comment:
I think you can have both `@OptionArgument` and `@Option` if you want to support `--tap=foo` and `--tap`. At least that's how `CeylonCompileTool` seems to do it.